tests/tests_configure_ha_cluster.yml

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Verify the role on the primary node
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_standard_eula: true
+    mssql_password: "p@55w0rD"
+    mssql_edition: Evaluation
+    mssql_ha_db_name: ExampleDB
+  tasks:
+    # - name: Gather system services facts
+    #   service_facts:
+
+    # - name: Open firewall ports for TCP connection to SQL Server
+    #   vars:
+    #     firewall:
+    #       - port: "{{
+    #           '1433' if mssql_tcp_port is not defined else mssql_tcp_port
+    #           }}/tcp"
+    #         zone: public
+    #         state: enabled
+    #         permanent: true
+    #         runtime: true
+    #     when: >-
+    #       ('running' in
+    #       ansible_facts['services']['firewalld.service']['state']) or
+    #       ('enabled' in
+    #       ansible_facts['services']['firewalld.service']['status'])
+    #   include_role:
+    #     name: fedora.linux_system_roles.firewall
+    - name: Skip this test on EL < 7 because ha_cluster does not support it
+      meta: end_play
+      when:
+        - ansible_distribution_version is version('8.3', '<')
+        - ansible_distribution in ['CentOS', 'RedHat']
+
+    - name: Set the mssql_ha_replica_type fact to appear in hostvars
+      set_fact:
+        mssql_ha_replica_type: primary
+      when: ansible_play_hosts_all |
+            map('extract', hostvars, 'mssql_ha_replica_type') |
+            list is not defined
+
+    - name: Set up test environment for the ha_cluster role
+      include_role:
+        name: fedora.linux_system_roles.ha_cluster
+        tasks_from: ../../../tests/ha_cluster/tasks/setup_test.yml
```

Comment:
I don't like using relative paths like this - what if we change the ha_cluster role to have a `ha_cluster_setup_for_test: true` and `ha_cluster_setup_sbd_for_test: true` parameters?  Then you could use
```yaml
    - name: Set up test environment and SBD for the ha_cluster role
      include_role:
        name: fedora.linux_system_roles.ha_cluster
        vars:
          ha_cluster_setup_for_test: true
          ha_cluster_setup_sbd_for_test: true
```
What you have is OK for now, but we should change this in the future.

Reply:
I've implemented your suggestion in https://github.com/linux-system-roles/ha_cluster/pull/50